docs/provisioning/silo_vnet_existingstorage.md

```diff
@@ -0,0 +1,93 @@
+# Create a silo with compute behind a vnet accessing your existing storage account
+
+This tutorial applies in the case you want to create a compute to process data from an existing storage account located in the same tenant (different subscription or resource group).
```

Comment:
Maybe add something like: "Typically, this is what would happen when the _internal_ silos correspond to various regions."